tests/Docker/Dockerfile

```diff
@@ -0,0 +1,27 @@
+ARG PHP_VERSION
+FROM --platform=linux/amd64 php:${PHP_VERSION}-cli-alpine
+
+# Install main package
+RUN apk add git libzip-dev autoconf g++ make linux-headers
+
+# Install PHP extensions
+RUN docker-php-ext-install zip sockets
+
+# Install PHP extensions from PECL
+RUN pecl install xdebug pcov \
+    && docker-php-ext-enable xdebug pcov
+
+# Install Composer
+COPY --from=composer:latest /usr/bin/composer /usr/local/bin/composer
+ENV COMPOSER_ALLOW_SUPERUSER 1
+
+# Install RoadRunner
+COPY --from=spiralscout/roadrunner:2023.1.4 /usr/bin/rr /usr/local/bin/rr
+
+# Set up work directory
+ADD . /app
+WORKDIR /app
+
+RUN composer update --prefer-dist --no-interaction
+
+CMD ["sleep", "infinity"]
```

Comment:
For what reason?

Reply:
Until no processes start inside. Added temporarily to keep the container running

Reply:
Still don't understand why, I'm sorry.

Isn't it better to run `/bin/sh` or `/bin/bash` if available at least?